src/main/resources/public/js/getContributor.js

```diff
@@ -270,21 +254,23 @@ function markContractForRemoval(contract, removeButton) {
         + "/mark?role=" + contract.id.role,
         {
             type: "DELETE",
-            success: function(marked) {
+            success: function (marked) {
                 var toolTipMessage = "This contract has been marked for removal on " + marked.markedForRemoval + ". "
                     + "No more tasks will be assigned to it and it will be removed after 30 days. "
                     + "Contact the PO if you want to have it reactivated."
                 removeButton.replaceWith("<i class='fa fa-exclamation-circle fa-lg' style='color:red;' aria-hidden='true' "
                     + "data-toggle='tooltip' data-placement='top' "
                     + "data-original-title='" + toolTipMessage + "'>"
-                    +"</i>");
-                $('[data-toggle="tooltip"]').tooltip();
+                    + "</i>");
+                $('[data-toggle="tooltip"]').tooltip({
+                    boundary: 'window'
+                });
```

Comment:
There  was small bug with the "marked for removal" tooltip arrow pointing in the wrong direction (it can be seen in the screenshot). This fixes it.